piwik/piwik.php

```diff
@@ -69,16 +70,21 @@ function piwik_analytics(string &$b)
 	$siteid  = DI::config()->get('piwik', 'siteid');
 	$optout  = DI::config()->get('piwik', 'optout');
 	$async   = DI::config()->get('piwik', 'async');
+	$shortendpoint = DI::config()->get('piwik', 'shortendpoint');
 
 	/*
 	 *   Add the Piwik tracking code for the site.
 	 *   If async is set to true use asynchronous tracking
 	 */
+	
+	$scriptAsyncValue = $async ? 'true' : 'false';
+	$scriptPhpEndpoint = $shortendpoint ? 'js/' : 'piwik.php';
+	$scriptJsEndpoint = $shortendpoint ? 'js/' : 'piwik.js';
+
+	$b .= "<!-- Piwik --> <script type=\"text/javascript\"> var _paq = _paq || []; _paq.push(['trackPageView']); _paq.push(['enableLinkTracking']); (function() { var u=((\"https:\" == document.location.protocol) ? \"https\" : \"http\") + \"://".$baseurl."\"; _paq.push(['setTrackerUrl', u+'$scriptPhpEndpoint']); _paq.push(['setSiteId', ".$siteid."]); var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript'; g.defer=true; g.async=$scriptAsyncValue; g.src=u+'$scriptJsEndpoint'; s.parentNode.insertBefore(g,s); })(); </script> <!-- End Piwik Code -->\r\n";
```

Comment:
Since we're using double-quotes, you can directly inline variables.
```suggestion
	$b .= "<!-- Piwik --> <script type=\"text/javascript\"> var _paq = _paq || []; _paq.push(['trackPageView']); _paq.push(['enableLinkTracking']); (function() { var u=((\"https:\" == document.location.protocol) ? \"https\" : \"http\") + \"://$baseurl\"; _paq.push(['setTrackerUrl', u+'$scriptPhpEndpoint']); _paq.push(['setSiteId', $siteid]); var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript'; g.defer=true; g.async=$scriptAsyncValue; g.src=u+'$scriptJsEndpoint'; s.parentNode.insertBefore(g,s); })(); </script> <!-- End Piwik Code -->\r\n";
```